lib/eex/lib/eex/engine.ex

```diff
@@ -133,11 +144,23 @@ defmodule EEx.Engine do
     end
   end
 
+  def handle_expr(_buffer, "/", _expr) do
```

Comment:
Please do not use `~w` when the "words" in it are a bit *weird* like in this case. I think `marker in ["/", "|"]` looks much more comfortable here.